.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: false
 language: node_js
 node_js:
-  - "4.1"
+  - "6.1"
```

Comment:
I was an idiot with widgets!  Upon debating it with @novemberborn I realised that it was best to use `6` with Travis. 😞